moment.js

```diff
@@ -2261,6 +2261,10 @@
         },
 
         fromNow : function (withoutSuffix) {
+            if (!this.isValid()) {
+                return this.localeData().invalidDate();
+            }
+
             return this.from(moment(), withoutSuffix);
```

Comment:
Shouldn't this change just go in `from` and be propagated out of there? Two birds, one stone.